src/main/webapp/app/tutor-exercise-dashboard/tutor-exercise-dashboard.component.ts

```diff
@@ -202,6 +209,8 @@ export class TutorExerciseDashboardComponent implements OnInit {
             submissionsObservable = this.textSubmissionService.getTextSubmissionsForExercise(this.exerciseId, { assessedByTutor: true });
         } else if (this.exercise.type === ExerciseType.MODELING) {
             submissionsObservable = this.modelingSubmissionService.getModelingSubmissionsForExercise(this.exerciseId, { assessedByTutor: true });
+        } else if (this.exercise.type === ExerciseType.FILE_UPLOAD) {
```

Comment:
Please also use a switch case for the type checks in this component

Reply:
Added switch cases to type checks in this file